src/ValueHandler/ProductOptionValueHandler.php

```diff
@@ -95,13 +161,85 @@ public function handle($productVariant, string $optionCode, array $akeneoValue):
                         $productVariant->getCode(),
                         $product->getCode(),
                         $optionCode,
-                        $partialValueCode
+                        $akeneoValue
                     )
                 );
             }
 
             throw $e;
         }
+        foreach ($akeneoAttributeOption['labels'] as $localeCode => $label) {
+            if ($this->translationLocaleProvider !== null &&
+                !in_array($localeCode, $this->translationLocaleProvider->getDefinedLocalesCodes(), true)) {
+                continue;
+            }
+            $optionValueTranslation = $optionValue->getTranslation($localeCode);
+            if ($optionValueTranslation->getLocale() !== $localeCode) {
+                /** @var ProductOptionValueTranslationInterface $optionValueTranslation */
+                $optionValueTranslation = $this->productOptionValueTranslationFactory->createNew();
+                $optionValueTranslation->setLocale($localeCode);
+            }
+            $optionValueTranslation->setValue($label);
+            if (!$optionValue->hasTranslation($optionValueTranslation)) {
+                $optionValue->addTranslation($optionValueTranslation);
+            }
+        }
+        if (!$productVariant->hasOptionValue($optionValue)) {
+            $productVariant->addOptionValue($optionValue);
+        }
+    }
+
+    private function handleMetricOption(ProductOptionInterface $productOption, string $optionCode, array $akeneoDataValue, ProductInterface $product, ProductVariantInterface $productVariant): void
+    {
+        if (!array_key_exists('amount', $akeneoDataValue)) {
+            throw new LogicException('Amount key not found');
+        }
+        $floatAmount = (string) ($akeneoDataValue['amount']);
+        if (!array_key_exists('unit', $akeneoDataValue)) {
+            throw new LogicException('Unit key not found');
+        }
+        $unit = (string) $akeneoDataValue['unit'];
+        $optionValueCode = $this->createOptionValueCode($optionCode, $floatAmount, $unit);
+
+        $optionValue = $this->getOrCreateProductOptionValue($optionValueCode, $productOption);
+
+        /** @var string[] $locales */
+        $locales = $this->getLocaleCodes($product);
+
+        foreach ($locales as $localeCode) {
+            $label = $floatAmount . ' ' . $unit;
+            if ($this->translator !== null) {
+                $label = $this->translator->trans('webgriffe_sylius_akeneo.ui.metric_amount_unit', ['unit' => $unit, 'amount' => $floatAmount], null, $localeCode);
+            }
```

Comment:
```suggestion
            $label = $this->translator->trans('webgriffe_sylius_akeneo.ui.metric_amount_unit', ['unit' => $unit, 'amount' => $floatAmount], null, $localeCode);
```

Reply:
Yes, but I will keep this kind of changes on the #130.

Reply:
Ok!